fdbrpc/FlowTransport.actor.cpp

```diff
@@ -399,7 +399,7 @@ struct ConnectPacket {
 		serializer(ar, connectPacketLength);
 		if (connectPacketLength > sizeof(ConnectPacket) - sizeof(connectPacketLength)) {
 			ASSERT(!g_network->isSimulated());
-			TraceEvent("SerializationFailed").backtrace();
+			TraceEvent("SerializationFailed").detail("PacketLength", connectPacketLength).backtrace();
```

Comment:
If we are reporting "connectPacketLength" then we might want to report "sizeof(ConnectPacket) - sizeof(connectPacketLength)" too (or the difference between them).

Reply:
sizeof(ConnectPacket) - sizeof(connectPacketLength) == 44 - 4. Both are constant.

Reply:
I see. Sorry, I missed that.